src/commands/suggestions/utils.ts

```diff
@@ -1,14 +1,13 @@
+import _ from 'lodash';
 import { suggestionStatesReadable, getAvailableStatesString } from '../../components/suggestions';
 
 export const validateIDs = (value: string): boolean | string => {
   // Split by white space
-  const values = value.split(' ');
+  const values = value.split(' ').map((a) => Number(a));
```

Comment:
why do we need to map each value to a number here? I thought `isNaN` below will just tell us if a string can't be parsed into a number?

Reply:
![image](https://user-images.githubusercontent.com/37851975/126334635-7c00f6a9-a6bd-44c4-ae0a-ae1d41e24dec.png)
Might be a typescript thing